### PR TITLE
New package: tmuxer-2.3.0

### PR DESCRIPTION
**Pull request by atweiden**

### CODE REVIEW

File: srcpkgs/tmuxer/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'tmuxer'
+pkgname=tmuxer
+version=2.2.0
+revision=1
+archs=noarch
+build_style=fetch
+depends="bash coreutils openssh sed tmux"
+short_desc="Tmux layout generator and command executor"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="MIT"
```
Comment (ndowens) [inline srcpkgs/tmuxer/template]: With MIT, you need to install a license with vlicense

**Comment by xtraeme** [comment]
Why do we need such scripts as xbps packages? I'm seeing lots of them being added lately...

File: srcpkgs/tmuxer/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'tmuxer'
+pkgname=tmuxer
+version=2.3.0
+revision=1
+archs=noarch
+depends="bash coreutils openssh sed tmux"
```
Comment (Piraty) [inline srcpkgs/tmuxer/template:6]: i doubt it hard depends on openssh

**Comment by sgn** [comment]
It looks like a simple shell script, can be installed to home bin.
I tempted to reject.
@Piraty opinions?

File: srcpkgs/tmuxer/template (hunk 2 above)
Comment (atweiden) [reply srcpkgs/tmuxer/template:6]: [It doesn't run without `ssh` being detected in $PATH]

See:

- https://github.com/tjhop/tmuxer/blob/master/tmuxer#L11
- https://github.com/tjhop/tmuxer/blob/master/tmuxer#L114

**Comment by atweiden** [comment]
@sgn note tmux-xpanes is in this repo already.

**Comment by atweiden** [comment]
I'm closing my open PRs in this repo for two reasons:

1. They don't seem to be going anywhere
2. There are now over 550 open pull requests

I've opined for quite some time now about how unsustainable it is to have one monorepo serving all of Void. As the PRs continue to pile up, the sustainability of the current approach becomes more and more questionable.

**Pull request closed without merging**